package.json

```diff
@@ -24,10 +24,17 @@
       "default": "./dist/index.js"
     },
     "./react": {
+      "types": "./dist/react/index.d.ts",
       "import": "./dist/react/index.js",
       "require": "./dist/cjs/react/index.js",
       "default": "./dist/react/index.js"
     },
+    "./react/media-store": {
```

Comment:
This allows us to directly import the react hooks et al. module without including `/dist` in the pathing for most modern build setups (tl;dr - they need to "know about" [package entry paths/"exports"](https://nodejs.org/api/packages.html#package-entry-points)). There are examples of this in the updated examples applications.